index.html

```diff
@@ -6,45 +6,75 @@
     <link rel="stylesheet" type="text/css" href="styles.css">
     <div class="t_b">
```

Comment:
Why you added HTML elements in <head> part ? 
you should not add any HTML elements in the head, you have to refer only script/css.